### DISCUSSION TITLE
What is the meaning of __p:false after converted JSON for css and how to remove # from id attribute in json file?

### DISCUSSION THREAD

**Discussion by senthil177cse** [GrapesJS/grapesjs#4278] [Q&A]
first of thanks to this amazing project @artf ...
how to remove any one of property in JSON file and what is the meaning of __p: false after converted JSON of css? for ref below is my JSON pasted..

[
  {
    "selectors": ["section"],
    "style": {
      "width": "100%",
      "__p": false,
      "height": "100vh",
      "padding-top": "15px",
      "padding-bottom": "15px",
      "padding-right": "15px",
      "padding-left": "15px"
    },
    "mediaText": "(max-width: 480px)",
    "atRuleType": "media"
  },
  {
    "selectors": ["#i5jl"],
    "style": {
      "color": "blue",
      "display": "flex",
      "__p": false,
      "margin-top": "10px",
      "margin-bottom": "0",
      "margin-right": "auto",
      "margin-left": "auto"
    },
    "mediaText": "(max-width: 480px)",
    "atRuleType": "media"
  },
  {
    "selectors": ["#i01j"],
    "style": { "color": "#00a187", "__p": false, "font-weight": "600" },
    "mediaText": "(max-width: 480px)",
    "atRuleType": "media"
  },
  {
    "selectors": ["form-control"],
    "style": { "margin-bottom": "10px", "__p": false },
    "mediaText": "(max-width: 480px)",
    "atRuleType": "media"
  },
  {
    "selectors": ["#ioef"],
    "style": { "font-weight": "600", "__p": false, "color": "#00a187" },
    "mediaText": "(max-width: 480px)",
    "atRuleType": "media"
  },
  {
    "selectors": ["btn"],
    "style": {
      "display": "flex",
      "__p": false,
      "margin-top": "0",
      "margin-bottom": "0",
      "margin-right": "auto",
      "margin-left": "auto",
      "background-color": "#00c0ca",
      "color": "#fbfbfb",
      "font-weight": "500"
    },
    "mediaText": "(max-width: 480px)",
    "atRuleType": "media"
  }
]

**Comment by artf** [comment]
Check the [CSS Composer API](https://grapesjs.com/docs/api/css_composer.html) in order to manage rules in your project.
The `__p` is an internal property, I'll remove it from the JSON in the next release.

**Reply by senthil177cse** [reply]
Hi @artf  still its not remove in ur latest release... __p: flase
